mirrormaker_replication_flow.go

```diff
@@ -11,17 +11,17 @@ type (
 
 	// ReplicationFlow a replication flow entity
 	ReplicationFlow struct {
-		Enabled                         bool     `json:"enabled"`
+		Enabled                         bool     `json:"enabled,omitempty"`
 		OffsetSyncsTopicLocation        string   `json:"offset_syncs_topic_location,omitempty"`
 		SourceCluster                   string   `json:"source_cluster,omitempty"`
 		TargetCluster                   string   `json:"target_cluster,omitempty"`
```

Comment:
These two fields are required https://api.aiven.io/doc/#tag/Service:_Kafka_MirrorMaker/operation/ServiceKafkaMirrorMakerCreateReplicationFlow

Reply:
Or it used as all-in-one object?

Reply:
yes, it's used in both request and response